### DISCUSSION TITLE
Emissive textures

### DISCUSSION THREAD

**Discussion by MattHopes** [MoreMcmeta/core#102] [Help Wanted]
Hello, I'm having a problem with a resourcepack with emissive compass which works fine for 1.20.1 fabric but not for 1.20.1 forge, got any solution?

**Comment by soir20** [comment]
Hi, could you please add more detail to the question? For examples, screenshots of what's working and not working would be helpful, as well as an upload or link to the resource pack/mods. I'm not aware of an existing issue so can't suggest solutions right off the bat.

**Reply by MattHopes** [reply]
https://github.com/user-attachments/assets/af1ff52f-813e-4392-91b9-9c01b7de3675

That's my pack but if needed I can upload that.
I'm using:
Forge 1.20.1 - Loader version 47.3.11
MoreMcmeta v1.20.1-4.4.7-forge
Moremcmeta Emissive Textures v1.20.1-2.0.4-forge

**Reply by MattHopes** [reply]
It seems like only one frame of the item works, infact items which have no animated textures work fine

**Reply by MattHopes** [reply]
Hello?

**Reply by soir20** [reply]
I haven't had a chance to investigate this yet as I've been working on the 4.5.0 release and other bug fixes

**Reply by MattHopes** [reply]
Okok thanks, let me know if you have any news!
